cmd/main.go

```diff
@@ -105,8 +105,8 @@ func main() {
 
 func initializeAPIRouter(feeCalculator *fees.Calculator) *apirouter.APIRouter {
 	apiRouter := apirouter.NewAPIRouter()
-	apiRouter.AddV1Router(metadata.NewMetadataRouter(feeCalculator))
-
+	apiRouter.AddV1Router(metadata.MetadataRoute, metadata.NewMetadataRouter(feeCalculator))
+	apiRouter.AddV1Router(healthcheck.HealthCheckRoute, healthcheck.NewHealthCheckRouter())
```

Comment:
the name of the packages are `healthcheck` and `metadata` therefore you do not need the redundant naming of `NewHealthCheckRouter` or `NewMetadataRouter. You can simply name it `NewRouter`
The code would look like:
```Go
apiRouter.AddV1Router(metadata.Route, metadata.NewRouter(feeCalculator))
apiRouter.AddV1Router(healthcheck.Route, healthcheck.NewRouter())
```